### ISSUE TITLE
python support update --> 3.12

### ISSUE THREAD

**Issue by pp-mo** [ocefpaf/cf_units#449]

**Comment by echoix** [comment]
Python 3.13 is just 10 days away now

**Comment by ocefpaf** [comment]
BTW, conda-forge is shipping a py313 version of it and all the tests are passing. You can test it out with:

```
conda env create --name CFUNITS python=3.13 cf-units pytest -c conda-forge/label/python_rc
conda activate CFUNITS
pytest --pyargs cf_units
```

**Comment by trexfeathers** [comment]
We should now be in a good position to add this without missing a spot:

https://github.com/SciTools/cf-units/blob/539dbccd5597b73c1785bbd31d51e4c304e7ffc1/cf_units/tests/test_coding_standards.py#L121-L122

**Comment by pp-mo** [comment]
Closed by #461
